README.md

```diff
@@ -114,6 +114,10 @@ code is a release candidate (rc) for the next version of Egeria (2.7.0 in this e
 
 Note that the 2 Egeria React UI npm packages are not, at this time, published to NPM.  
 
+## SSL configuration
+
+By default the Egeria React UI uses a truststore.p12 and keystore.p12 files for ssl. The p12 files are copies of files 'https://github.com/odpi/egeria/blob/master/keystore.p12' and 'https://github.com/odpi/egeria/blob/master/truststore.p12', which are the definitive sources of these files. The keystore and trustore files allow Egeria to run simply in a demo/development; this is not appropriate for production, which should be appropriately secured.     
```

Comment:
> I think this is a sensible starting point -- when we make changes in the base with how we store or deploy the certs we should change this.
> 
> There is another slight challenge - the egeria react UI has multiple roles
> 
> * It is a client of, and makes connections TO the egeria base server. Thus it needs to be able to check the cert that server provides (typically via a CA in what we'd call the truststore - could also be a cert in itself). Optionally it may also need to present a cert to the backend if using 2 way SSL. In this case it needs a certificate (probably from the keystore), as well as an identifier (if using a store, to determine which cert to use). In both cases a keystore password is also needed
>   BUT
> * it is in itself a server -- or at least runs within a serving container. I know little about this environment, but the same concept must exist. It needs a certificate to present to it's users (the browser), and may need a CA/cert to check the browser's certs against when using 2 way SSL.
> 
> So the words truststore and keystore can get confusing as we have multiple contexts. It's also certainly the case we should allow for each of these to be independent, since it might be real wild external users coming in from the UI, but a much more secure and restricted check at the backend (and here 2 way might be more likely)
> 
> So I think the action here is to allow each of these to be independent. I'm happy to try and figure out how to write down the text better?

Also truststore and keystore are java terms - so do not make any sense to a nodejs person....

Reply:
Good observation :-) We'll need some mapping language between the two. Basically both contain keys and/or certificates (and/or certificate authorities).

So when we say 'keystore' we're talking around the key that gets sent by the app & all the things that go with it - be it a raw cert, a .p12 file, with a password, and maybe an index label, whilst for truststore we just mean the single certificate (unlikely) or rather than file containing a set of certificates/CAs that are used to trust information we received :-)